<launch>
    <arg name="agent_name" default="agentA1" />

    <node pkg="rhbp_workspace" type="rhbp_agent.py"
        name="rhbp_agent_$(arg agent_name)" output="screen"
        >
            <param name="agent_name" value="$(arg agent_name)" />
    </node>

    <!-- RHBP manager configuration is separated from the node
         because it uses a special namespace 'prefix/rhbp_manager'
         Limits of parameters which are accessible in dynamic_reconfigure are set in cfg/Manager.cfg
         -->
    <group ns="$(arg agent_name)/rhbp_manager">
        <param name="situationBias" type="double" value="1.0" />
        <param name="predecessorBias" type="double" value="1.0" />
        <param name="successorBias" type="double" value="1.0" />
        <param name="conflictorBias" type="double" value="1.0" />
        <param name="goalBias" type="double" value="1.0" />
        <param name="planBias" type="double" value="1.0" />
        <param name="activationThreshold" type="double" value="7.0" />
        <param name="activationGoalPriority" type="bool" value="True"/>
        <param name="createLogFiles" type="bool" value="true"/>
        <param name="activation_algorithm" type="string" value="default"/>
        <param name="max_parallel_behaviours" type="int" value="1"/> <!--Comment or negative for unlimited-->
        <param name="plan_monitoring_all_sensor_changes_by_behaviours" type="bool" value="false"/>
        <param name="plan_monitoring_behaviour_missing_influence" type="bool" value="true"/>
        <param name="plan_monitoring_unexpected_behaviour_finished" type="bool" value="true"/>
    </group>

</launch>
